### DISCUSSION TITLE
Location of dependency files directory

### DISCUSSION THREAD

**Discussion by MKatereggaExigent** [marcelotduarte/cx_Freeze#1265] [Q&A] [answered]
I have been able to build the exe folder but when I run the executable, it looks for dep files in my home dir instead of the built dir. How can I adjust my setup.py so that the build .exe file searches within the build dir when running.

Here is my setup.py

`from cx_Freeze import setup, Executable`

`build_options_exe = {'include_files':['general_report_notes.xlsx','iso_4217_currency_codes.csv']}`
`build_options_app = {'iconfile': 'mls.ico','bundle_name': 'FCPSoftwareapp'}`
`build_options_dmg = {'volume_label': 'FCPSoftwareapp'}`

`import sys`
`base = 'Win32GUI' if sys.platform=='win32' else None`

`executables = [`
`    Executable('FCPSoftware.py', base=base, target_name = 'FCPSoftware', icon="mls.ico")`
`]`

`setup(name='FCPSoftware',`
`      version = '4.0',`
`      description = 'Test application.',`
`      options = {'build_exe': build_options_exe, 'bdist_mac': build_options_app, 'bdist_dmg': build_options_dmg},`
`      executables = executables)`

**Comment by marcelotduarte** [comment, marked as answer]
See if this information helps you: https://cx-freeze.readthedocs.io/en/latest/faq.html#using-data-files

**Reply by MKatereggaExigent** [reply]
Thank you @marcelotduarte this worked straight away. You are a star!
